.github/workflows/main.yml

```diff
@@ -160,6 +160,7 @@ jobs:
               networkx==2.5
               requests==2.22.0
               semantic_version==2.8.4
+              PyGithub==1.54.1
```

Comment:
Please change version to 1.45, it seems to be standard on current reference version of Fedora.

Reply:
I made a similar comment in the `ci` pull request. 1.54 is the latest available with pip.